src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,6 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return 'the responses are identical';
```

Comment:
i see. hm, there is no verbosity level we can check, right? then lets keep
like this. i was thinking about a miss because its an error response and
similar problems